### PR TITLE
Prepare Release 1.9.0

### PR DESCRIPTION
**Pull request by bitPogo**
Prepare release of 1.9.0

### CODE REVIEW

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -12,33 +12,41 @@ and this project adheres to http://semver.org/spec/v2.0.0.html[Semantic Versioni
...
 * D4LSDKFlow (util-coroutine)
 * util-coroutine-test module
 * util-ktor-test module
 * HttpMockObjectResponse (util-ktor-test)
 * createHelloWorldOkResponse, createHelloWorldMockClient, createErrorMockClient and createMockClientWithResponse (util-ktor-test)
+* stately (1.1.6)
 
 === Changed
 
 * _**BREAKING**_ runBlockingTest and runWithContextBlockingTest are now part of util-coroutine-test
```
Comment (alessioborraccino) [inline CHANGELOG.adoc:44]: these might need some migratino instruction?

File: CHANGELOG.adoc [hunk 2]
```diff
@@ -12,33 +12,41 @@ and this project adheres to http://semver.org/spec/v2.0.0.html[Semantic Versioni
...
 === Added
 
 * CommonResourceLoad (util-test)
-* HttpMockObjectResponse (util-test)
 * util-coroutine module
 * D4LSDKFlow (util-coroutine)
 * util-coroutine-test module
 * util-ktor-test module
 * HttpMockObjectResponse (util-ktor-test)
 * createHelloWorldOkResponse, createHelloWorldMockClient, createErrorMockClient and createMockClientWithResponse (util-ktor-test)
```
Comment (alessioborraccino) [inline CHANGELOG.adoc:39]: These ones are now scoped

**Review by alessioborraccino — CHANGES_REQUESTED** [review]

File: CHANGELOG.adoc (hunk 2 above)
Comment (bitPogo) [reply CHANGELOG.adoc]: Done

File: CHANGELOG.adoc (hunk 1 above)
Comment (bitPogo) [reply CHANGELOG.adoc:45]: Not that I know of, since it completely new and only relates to util-coroutine...but the latter is worth to mention

Comment (bitPogo) [reply CHANGELOG.adoc:45]: done

**Review by alessioborraccino — APPROVED** [review]

**Pull request merged**